go/integTest/shared_commands_test.go

```diff
@@ -4457,6 +4457,241 @@ func (suite *GlideTestSuite) TestZRem() {
 	})
 }
 
+func (suite *GlideTestSuite) TestZRange() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		key := uuid.New().String()
+		memberScoreMap := map[string]float64{
+			"a": 1.0,
+			"b": 2.0,
+			"c": 3.0,
+		}
+		_, err := client.ZAdd(key, memberScoreMap)
+		assert.Nil(suite.T(), err)
```

Comment:
```suggestion
		assert.Nil(t, err)
```
Similarly for all occurrences of this in both tests.